Cognite.Jetfire.Cli/Show/ShowCommand.cs

```diff
@@ -75,5 +94,84 @@ async Task Handle(string cluster, int? id, string externalId)
+            {
+                int configId = await Utils.ResolveEitherId(id, externalId, client);
+                var jobs = await client.TransformConfigRecentJobs(configId);
+
+                var filteredJobs = jobs.Where(job => job.Uuid == jobId);
+                if (filteredJobs.Count() == 0)
+                {
+                    throw new JetfireCliException($"Job with UUID {jobId} not found");
+                }
+
+                var job = filteredJobs.First();
+
+                Console.WriteLine($"Job UUID:            {job.Uuid}");
+                Console.WriteLine($"Transformation ID:   {configId}");
+
+                Console.WriteLine();
+
+                Console.WriteLine($"Destination:         {job.DestinationType}");
+                if (job.DestinationType == "raw_table")
+                {
+                    Console.WriteLine($"   Database:        {job.DestinationDatabase}");
+                    Console.WriteLine($"   Table:           {job.DestinationTable}");
+                }
+                Console.WriteLine($"Destination project: {job.DestinationProject}");
+                Console.WriteLine($"Action:              {job.ConflictMode}");
+
+                Console.WriteLine();
+
+                Console.WriteLine($"Created at:          {Utils.FormatTimestamp(job.CreatedTime)}");
+                Console.WriteLine($"Started at:          {Utils.FormatTimestamp(job.StartedTime)}");
+                Console.WriteLine($"Finished at:         {Utils.FormatTimestamp(job.FinishedTime)}");
+                Console.WriteLine($"Duration:            {Utils.FormatDuration(job.StartedTime, job.FinishedTime)}");
+                Console.WriteLine($"Status:              {job.Status()}");
+                if (job.Error != null)
+                {
+                    Console.WriteLine($"   Error:            {job.Error}");
+                }
+
+                Console.WriteLine();
+                Console.WriteLine("Progress:");
+                var metrics = await GetLatestMetrics(jobId, client);
+                if (metrics.Count > 0)
+                {
+                    foreach (var metric in metrics)
+                    {
+                        Console.WriteLine($"   {metric.Key,-17} {metric.Value}");
+                    }
+                }
+                else
+                {
+                    Console.WriteLine("   No progress available");
+                }
+
+
+                Console.WriteLine();
+                Console.WriteLine("".PadLeft(50, '-'));
+                Console.WriteLine();
+
+                Console.WriteLine(job.RawQuery);
+            }
+
+            async Task<Dictionary<string, long>> GetLatestMetrics(string jobId, IJetfireClient client)
+            {
+                var metrics = await client.TransformJobMetrics(jobId, new CancellationToken());
+                var dict = new Dictionary<string, long>();
+
+                // Assume sorted result from API
```

Comment:
I would not make that assumption if I were you :)

Reply:
Ah, It seemed sorted when I looked at it, but fair. I'll redo it properly 🙂